src/marisa_trie.pyx

```diff
@@ -248,33 +254,37 @@ cdef class _Trie:
         self._trie.mmap(c_path)
         return self
 
-    def iterkeys(self, unicode prefix=""):
+    def iterkeys(self, prefix=NoPrefixGiven):
```

Comment:
I think we should just go with `None` then. 

I'd be happy to merge this once the tests are green. The current failure seems to be caused by the outdated C++ source. Please recompile them using `update_cpp.sh`.